pkg/restapi/issuer/operation/models.go

```diff
@@ -16,6 +16,8 @@ import (
 )
 
 // ProfileDataRequest req for profile creation.
+// Issuer ID identifies who is the issuer of the credential manifests being issued.
+// CMStyle represents an entity styles object as defined in credential manifest spec.
```

Comment:
Can we add these field level specific comments at field level than at struct level ?